grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -354,33 +365,6 @@ class CycleCountService {
         )
     }
 
-    CycleCountDto updateCycleCount(CycleCount cycleCount, List<CycleCountItem> cycleCountItems, CycleCountStartCommand request) {
-        if (cycleCount.status in [CycleCountStatus.READY_TO_REVIEW, CycleCountStatus.CANCELED]) {
-            throw new IllegalArgumentException("Cycle count is already in review or canceled")
-        }
-        if (cycleCount.status in [CycleCountStatus.INVESTIGATING, CycleCountStatus.COUNTED]) {
-            throw new UnsupportedOperationException("Support will be added later")
-        }
-        if (cycleCount.status in CycleCountStatus.listInProgress()) {
-            CycleCountItem itemWithHighestCountIndex = cycleCountItems.max { it.countIndex }
-            if (itemWithHighestCountIndex?.countIndex != request.countIndex) {
-                throw new IllegalArgumentException("Count index can't be higher than the highest count index of the items")
-            }
-        }
-        return CycleCountDto.toDto(cycleCount)
-    }
```

Comment:
what happened with that method? I can't find where it has been moved or why it has been removed